src/test/java/parsing/ParserTest.java

```diff
@@ -2,26 +2,17 @@
 
 import datastructure.Node;
 import datastructure.NodeGraph;
+import java.io.*;
+import java.lang.reflect.Method;
 import javafx.application.Platform;
+import org.junit.After;
+import org.junit.Test;
 import org.junit.runner.RunWith;
 import org.powermock.api.mockito.PowerMockito;
 import org.powermock.core.classloader.annotations.PrepareForTest;
 import org.powermock.modules.junit4.PowerMockRunner;
 
-import java.io.BufferedReader;
-import java.io.InputStreamReader;
-import java.io.FileInputStream;
-import java.io.BufferedWriter;
-import java.io.FileWriter;
-import java.io.IOException;
-import java.io.File;
-
-import org.junit.After;
-import org.junit.Test;
-
-import static org.junit.Assert.assertEquals;
-import static org.junit.Assert.assertTrue;
-import static org.junit.Assert.fail;
+import static org.junit.Assert.*;
```

Comment:
We should avoid * imports, you can turn off automatic * imports by  going to file -> settings ->Editor -> Code Style -> Java. Under the tab imports you can set "Class count to use imports with '*'" to a large number to avoid automatic * imports.